llama-index-core/llama_index/core/program/utils.py

```diff
@@ -90,9 +90,15 @@ def get_program_for_llm(
             **kwargs,
         )
     elif pydantic_program_mode == PydanticProgramMode.LM_FORMAT_ENFORCER:
-        from llama_index.core.program.lmformatenforcer_program import (
-            LMFormatEnforcerPydanticProgram,
-        )
+        try:
+            from llama_index.program.lmformatenforcer import (
+                LMFormatEnforcerPydanticProgram,
+            )
```

Comment:
Can we add the `  # pants: no-infer-dep` comment at the end of this import? (Similar to other 3rd party imports in core)

This helps avoid pants detecting this as a dependency when testing, making the venv smaller for core

Reply:
sorry just seeing this now. on it!

Reply:
that did the trick - all checks have passed now :)